### PR TITLE
IDR & Scientific Data announcement

### PR DESCRIPTION
**Pull request by hflynn**
Text taken from https://docs.google.com/document/d/1nRGgQVAS1a4Emeunh1njVWWm8Di-QLgH_oEfkdjehT0/edit#

Requires proofing only - text already cleared by Jason, Gabry & Varsha from Nature.

To be published just before 2pm on Monday ready for announcements

### CODE REVIEW

File: _posts/2018-03-26-idr.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: The Image Data Resource is added to the list of Scientific Data recommended data repositories
+categories: community
+---
+
+Nature Research journal [*Scientific Data*](https://www.nature.com/sdata/), is
```
Comment (mtbc) [inline _posts/2018-03-26-idr.md]: comma seems odd

File: _posts/2018-03-26-idr.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+mandates the release of the datasets that accompany their manuscripts, they do
+not themselves host data. Instead, they encourage submission of data to
+community-recognized data repositories where possible.
+
+The [Image Data Resource](https://idr.openmicroscopy.org/about/) (IDR) has
+just been listed, by *Scientific Data*, as the recommended data repository for
+bioimaging data, the first for this data type. Authors wishing to submit a
+data descriptor that includes imaging data will be asked to deposit the data
+in IDR, as part of the manuscript submission process. IDR accepts reference
+image datasets (as defined  at the
```
Comment (mtbc) [inline _posts/2018-03-26-idr.md]: double space

File: _posts/2018-03-26-idr.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+[IDR Submissions](https://idr.openmicroscopy.org/about/submission.html)).
+Datasets that are not reference images can be published using data archives
+(e.g. [BioStudies](https://www.ebi.ac.uk/biostudies/) or
+[Data Dryad](https://datadryad.org/)), or if appropriate, any of the other
```
Comment (mtbc) [inline _posts/2018-03-26-idr.md]: this comma I'd move after the "or"

**Comment by mtbc** [comment]
In this case my comments are but minor suggestions that can be ignored.

File: _posts/2018-03-26-idr.md (hunk 2 above)
Comment (will-moore) [reply _posts/2018-03-26-idr.md]: Suggestion: maybe remove 'the'?  ```(as defined at IDR Submissions)``` or, with 'the' ```(as defined at the IDR Submissions page)```

**Comment by will-moore** [comment]
Looks good.

**Pull request merged**